### PR TITLE
docs: add glossary page to KSD docs

### PR DESCRIPTION
**Pull request by gauravsitlani**
Signed-off-by: gauravsitlani <gauravsitlani@riseup.net>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://docs.koor.tech/docs/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://docs.koor.tech/docs/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/koor-tech/koor/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by zalsader — COMMENTED** [review]
Let's wait until the rook PR merges. Also Let's add a definition for KSD

**Comment by gauravsitlani** [comment]
Waiting for final approval on https://github.com/rook/rook/pull/11525 now.

**Comment by gauravsitlani** [comment]
Closing this for the meantime, will re-open soon once the other rook PR is merged.

**Comment by gauravsitlani** [comment]
docs-check failed due to : 
```
WARNING  -  Documentation file 'Getting-Started/glossary.md' contains a link to 'CRDs/Object-Storage/ceph-object-realm-crd.md' which is not found in the documentation files.
WARNING  -  Documentation file 'Getting-Started/glossary.md' contains a link to 'CRDs/Object-Storage/ceph-object-zonegroup-crd.md' which is not found in the documentation files.
WARNING  -  Documentation file 'Getting-Started/glossary.md' contains a link to 'CRDs/Object-Storage/ceph-object-zone-crd.md' which is not found in the documentation files.
WARNING  -  Documentation file 'Getting-Started/glossary.md' contains a link to 'Troubleshooting/krew-plugin.md' which is not found in the documentation files.
```

File: Documentation/Getting-Started/glossary.md
Comment (zalsader) [inline Documentation/Getting-Started/glossary.md:1]: Can we add a definition for Koor Storage Distribution (KSD)? Anything else Koor-specific? Maybe Extended Ceph Exporter?

Comment (gauravsitlani) [reply Documentation/Getting-Started/glossary.md:1]: Sure, sounds good. How do we want to define KSD ? I think we can leave Extended Ceph exporter out as it's monitoring project right rather than a terminology or a concept.

**Comment by gauravsitlani** [comment]
hey @galexrt and @ideepika please review and share your feedback as well.

**Review by ideepika — DISMISSED** [review]
looks good to me!

File: Documentation/Getting-Started/glossary.md
Comment (galexrt) [reply Documentation/Getting-Started/glossary.md:1]: @gauravsitlani I think mainly as a that KSD is a product based on Rook Ceph that makes running and maintaining Ceph storage on Kubernetes easier. What do you think?

I don't think we need to mention the extended-ceph-exporter as it is not mentioned/ used in the Koor docs (yet?), but it makes sense from a "promoting" the project perspective. In the future we should probably mention the extended-ceph-exporter in the monitoring documentation as well.

**Comment by gauravsitlani** [comment]
hey @galexrt I believe we're good to go ahead as soon as all the tests return green.

**Pull request closed without merging**
